docs/RestrictedPython4/index.rst

```diff
@@ -9,12 +9,11 @@ As the ``compiler`` module has been declared deprecated in Python 2.6 and alread
 Goals for a rewrite
 -------------------
 
-RestrictedPython is a core dependency for the Zope2 application server and therefore implicite for the content management system Plone.
-The Zope & Plone community want to continue there projects and as Python 2 is announced end of life by 2020, and will be ultimative replaced by Python 3 or higher.
+RestrictedPython is a core dependency for the Zope2 application server and therefore implicit for the content management system Plone.
+The Zope & Plone community want to continue their projects and as Python 2 is announced end of life by 2020, and will be ultimately replaced by Python 3 or higher.
 Zope and Plone should become Python 3 compatible.
```

Comment:
@jcbrand any help is welcome. You could also do the correction yourself immediatly and push to this branch; additionally we are in te #sprint channel